torch/signal/windows/windows.py

```diff
@@ -759,3 +732,71 @@ def general_hamming(M,
                           layout=layout,
                           device=device,
                           requires_grad=requires_grad)
+
+
+@_add_docstr(
+    r"""
+Computes the minimum 4-term Blackman-Harris window according to Nuttall.
+
+.. math::
+    w_n = 1 - 0.3635819 \cos{(z)} + 0.4891775 \cos{(2z)} - 0.1365995 \cos{(3z)} + 0.0106411 \cos{(4z)}
+
+where
+
+.. math::
+    z = \frac{2 \pi j}{M}, j = 0 \cdots M - 1
```

Comment:
```suggestion
where ``z_j = 2 π j / M``.
```
to save some vertical space.